test/api_integration/apis/saved_objects/bulk_get.ts

```diff
@@ -73,6 +73,7 @@ export default function ({ getService }: FtrProviderContext) {
                 },
                 coreMigrationVersion: '8.8.0',
                 typeMigrationVersion: resp.body.saved_objects[0].typeMigrationVersion,
+                managed: resp.body.saved_objects[0].managed,
```

Comment:
Same remark than previous comment. We're not asserting anything here. We need to update the dataset to properly assert the behavior.

Reply:
smae question as in https://github.com/elastic/kibana/pull/154515#discussion_r1165098941